packages/shared/src/components/modals/streaks/NewStreakModal.tsx

```diff
@@ -58,9 +58,8 @@ export default function NewStreakModal({
       trackEvent({
         event_name: AnalyticsEvent.DismissStreaksMilestone,
       });
+      completeAction(ActionType.DisableReadingStreakMilestone);
```

Comment:
Wait we use action for this, does it work with disabling it again?
Not sure we ever clear actions?

Reply:
This we can never disable, this is only about the modals. The streaks in general we're using settings - u can enable disable again. Discussion here https://dailydotdev.slack.com/archives/C05TQL6HWTH/p1717146432887299

Reply:
Oh ok this one is a once you remove it never get it back thing :)

Reply:
There's also no way ui wise even if it was with settings to get this back